app/controllers/recipes_controller.rb

```diff
@@ -13,14 +13,19 @@ def new
     @user = User.find(params[:user_id])
   end
 
+  def show
+    @user = User.find(params[:user_id])
+    @recipe = @user.recipes.find(params[:id])
+  end
+
   def create
     @recipe = Recipe.new(recipe_params)
     @recipe.user = User.find(params[:user_id])
     if @recipe.save
-      flash[:success] = t("success")
-      redirect_to "/"
+      flash[:success] = I18n.t('layouts.application.success')
```

Comment:
This is awesome that you're doing localization so early on in the process. One thing that stuck out to me is that this success message for creating a recipe is directly under the `application` key. That feels odd to me since this is an recipe not an _application_ being created.

How would you feel about instead changing this string to be under `recipes.create.success`? Bonus if you do that, you can shorten this to `t(".success")` since rails will know you're in the `create` action of the `recipes` controller.

Note: I had to look this up to remember where things should be placed and I found a perfect example [here](https://guides.rubyonrails.org/i18n.html#lazy-lookup) with the BooksController.